### ISSUE TITLE
How to apply a local quill.snow.css to an Ionic app?

### ISSUE THREAD

**Issue by JCMilito** [quilljs/quill#2963]
Please describe the a concise description and fill out the details below. It will help others efficiently understand your request and get to an answer instead of repeated back and forth. Providing a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) will further increase your chances that someone can help.

**Steps for Reproduction**

1. Visit [quilljs.com, jsfiddle.net, codepen.io]
2. Step Two
3. Step Three

**Expected behavior**:

**Actual behavior**:

**Platforms**:

Include browser, operating system and respective versions

**Version**:

Run `Quill.version` to find out

**Comment by JCMilito** [comment]
So, I'm developing an Ionic app.
When I include a tag < link rel="stylesheet" href="//cdn.quilljs.com/1.3.7/quill.snow.css" >, everything works fine.
Turns out that, I need it to work offline, and when I try < link rel="stylesheet" href="quill.snow.css" >, it gives me "Failed to load quill.snow.css", even if the file is the same folder as the html.
I even copied the contents from cdn.quilljs.com/1.3.7/quill.snow.css and pasted in my scss file, to no avail; all I got is a giant triangle where the editor should be.
What I; could be doing wrong?
